cnf-tests/testsuites/e2esuite/bond/bond_sriov.go

```diff
@@ -32,63 +31,69 @@ var _ = Describe("[sriov] Bond CNI integration", func() {
 	apiclient := client.New("")
 
 	execute.BeforeAll(func() {
+		if discovery.Enabled() {
+			Skip("Tuned sriov tests disabled for discovery mode")
```

Comment:
This should go into `BeforeEach`
`BeforeAll` will Skip only the first Context

Reply:
right! fixing ...